### DISCUSSION TITLE
Use fly cli to view status of resource check

### DISCUSSION THREAD

**Discussion by mcrapts** [concourse/concourse#7664] [Help & Support]
Using the fly CLI (`fly -t target builds`), I can see one of the resource check triggers failed. However, I am not sure how to view the log of why the resource check has failed. I can see the output using the UI, but I primarily use the fly CLI. `fly watch` only has the ability to watch jobs, not resource checks. Am I missing something or is this actually not possible using the fly CLI?

So the question is: how can I view the output of a resource check using the fly CLI?

**Comment by taylorsilva** [comment]
If you're on a post-7.x version then you should be able to watch checks which run as builds now with `fly watch --build=<build-number>` which you can get from `fly builds`.
